cmd/parca-agent/main.go

```diff
@@ -163,7 +164,7 @@ func main() {
 		os.Exit(1)
 	}
 
-	if flags.ExperimentalEnableDWARFUnwinding && runtime.GOARCH == "arm64" {
+	if !flags.DisableDWARFUnwinding && runtime.GOARCH == "arm64" {
```

Comment:
Perhaps we can suggest to the user what to do? In this case, we would recommend to add the `--disable-dwarf-unwinding` flag, right?

Or should we detect we're on ARM64 and turn it off? I kind of like this because it means when people deploy it fleet-wide (which is typical on Kubernetes) across amd64 and arm64 machines they can deploy them with consistent flags.

Reply:
Actually right now we should probably just exit on ARM64. With the improved frame pointer detection the Agent won't work on this architecture.

We need to add support for a bunch of things, including kernel tests for ARM64 and integration tests to ensure we don't regress

Reply:
hmm .. is there anything best effort we can do? We already know of some people who run the agent without DWARF unwinding enabled on arm64...